model_zoo/pytorch/nanogpt/prepare.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
1. Why do we not use the original [prepare.py](https://github.com/karpathy/nanoGPT/blob/master/data/shakespeare_char/prepare.py) in nanogpt?

2. We should list the link of the original prepare.py in nanogpt if we has referred to it.

Reply:
Thank you for your comment. Indeed, I have modified the original `prepare.py` script from NanoGPT to create a more generic data preparation process, which can be adapted to a wider range of datasets.

Reply:
I will also add this link to our guide.md file for better reference. Thanks for the suggestion!